pkg/client/monitoring/v1/alertmanager.go

```diff
@@ -177,6 +177,9 @@ func UnstructuredFromAlertmanager(a *Alertmanager) (*unstructured.Unstructured,
 	if err := json.Unmarshal(b, &r.Object); err != nil {
 		return nil, err
 	}
+	// TODO: See the corresponding comment in UnstructuredFromPrometheus.
```

Comment:
Would you mind adding the explanation comment here as well? I would prefer duplicate comments over comment referencing. Feel free to still keep a reference like `See 'unstructured.RemoveNestedField' in prometheus.go` here as well, so if we delete one, we keep in mind to delete the other as well.

Reply:
Done, and squashed